DotNet/proxy.ashx

```diff
@@ -1,4 +1,4 @@
-<%@ WebHandler Language="C#" Class="proxy" %>
+<asp:TreeView runat="server"></asp:TreeView><%@ WebHandler Language="C#" Class="proxy" %>
```

Comment:
@bsvensson Didn't intend to add TreeView. Hmm, maybe Visual Studio did that for me? Thanks for catching and removing it.